dali/pipeline/data/views.h

```diff
@@ -72,15 +73,22 @@ kernels::TensorShape<ndim> tensor_shape(const TensorList<Backend> &tl) {
   return out;
 }
 
+
 /// @brief Returns an equivalent tensor list shape for a tensor.
 ///        Outermost dimension is converted into sample index.
-template <int ndim, typename Backend>
+template<int ndim, typename Backend>
```

Comment:
Please capitalize first letter of template arguments

Reply:
I don't feel entitled to change it, since this code was subject of quite recent review and there are many places where this issue occur. I only added some error msg here ;)